src/gui/components/record/conflict/conflictDialog.tsx

```diff
@@ -0,0 +1,345 @@
+ * See, the License, for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Filename: RecordTabBar.tsx
+ * Description:
+ *   TODO
+ */
+
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import {styled} from '@mui/material/styles';
+import Dialog from '@mui/material/Dialog';
+import DialogTitle from '@mui/material/DialogTitle';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogActions from '@mui/material/DialogActions';
+import CloseIcon from '@mui/icons-material/Close';
+import Typography from '@mui/material/Typography';
+import IconButton from '@mui/material/IconButton';
+import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
+import {CircularProgress} from '@mui/material';
+import {Grid, Box} from '@mui/material';
+
+export function ConflictButton(props: any) {
+  return (
+    <Button
+      aria-label={props.text}
+      onClick={() => props.onButtonClick(props.id)}
+      value={props.value}
+      id={props.id}
+      variant={props.ischoose ? 'outlined' : 'contained'}
+      style={{height: '56px'}}
+      disabled={props.disabled}
+    >
+      {props.text}
+    </Button>
+  );
+}
+
+const ConflictDialog = styled(Dialog)(({theme}) => ({
+  '& .MuiDialogContent-root': {
+    padding: theme.spacing(2),
+  },
+  '& .MuiDialogActions-root': {
+    padding: theme.spacing(1),
+  },
+}));
+
+export interface DialogTitleProps {
+  id: string;
+  children?: React.ReactNode;
+  onClose: () => void;
+}
+
+const ConflictDialogTitle = (props: DialogTitleProps) => {
+  const {children, onClose, ...other} = props;
+
+  return (
+    <DialogTitle sx={{m: 0, p: 2}} {...other}>
+      {children}
+      {onClose ? (
+        <IconButton
+          aria-label="close"
+          onClick={onClose}
+          sx={{
+            position: 'absolute',
+            right: 8,
+            top: 8,
+            color: theme => theme.palette.grey[500],
+          }}
+        >
+          <CloseIcon />
+        </IconButton>
+      ) : null}
+    </DialogTitle>
+  );
+};
+
+function ConflictDialogContent() {
+  return (
+    <Box pb={20} pt={10} pl={3} pr={3}>
+      <Grid container spacing={{xs: 2, md: 3}}>
+        <Grid
+          item
+          xs={12}
+          container
+          justifyContent="center"
+          alignItems="flex-end"
+        >
+          <Typography>
+            The most recent conflict has been pre-selected. Choose another using
+            the dropdown box
+          </Typography>
+          <br />
+        </Grid>
+        <Grid
+          item
+          xs={6}
+          container
+          justifyContent="center"
+          alignItems="flex-end"
+        >
+          <CircularProgress />
+        </Grid>
+        <Grid item xs={6}>
+          <Typography>
+            Confilct is still downloading to this device, and it not yet
```

Comment:
@KateSHENG This requires grammar checking and a sign off by one of your peers before I continue this review.

Reply:
@Denubis leave this review with me, will take a look this afternoon. We're mainly focusing on the functionality as Kate gets the conflict interface up-and-running, but will try and catch typos as we go.

Reply:
Great, thanks! (I'm just worried that this will make it into the build we show clients as we get distracted by other bugs we find.) It might be worth using https://www.npmjs.com/package/i18n to start pulling out strings (just as we go, with an eye towards doing this properly fur july/august) for 
a) easier review
b) reduction of work for Kate, so she can focus on functionality
c) the ability to drop the strings file through Grammarly or equivalent. 
d) We're going to need dynamic i18n sooner rather than later, to match FAIMS 2.6 functionality, so this seems like a not horrible excuse to move towards it?

Downside: it's really late in the cycle to be adding things like that -- if you've got ideas about ways to do typo checking -- I suspect they'll be better than mine.